### PR TITLE
:bug: Firefox on iOS make Google Auth text readable

### PR DESCRIPTION
**Pull request by eugenebakaiev-okta**
Firefox on iOS make Google Auth text readable

Resolves: OKTA-124746

Reviewers:

@samerfanek-okta

### CODE REVIEW

**Comment by eugenebakaiev-okta** [comment]
Before: 
img_0006 jpg

After:
img_0005 jpg

File: assets/sass/modules/_enroll.scss [hunk 1]
```diff
@@ -403,6 +403,7 @@
 [name="sharedSecret"] {
   text-align: center;
   cursor: text;
+  opacity: 1;
```
Comment (hor-kanchan-okta) [inline assets/sass/modules/_enroll.scss]: `1` is default value. Is the opacity overridden? If so, where?

Comment (eugenebakaiev-okta) [inline assets/sass/modules/_enroll.scss]: This input is being located inside the span with class .o-form-disabled, which has a property opacity: 0.4. For some reason Firefox applied this opacity once again to this input which is direct sibling, so we have input with opacity 0.4 too. As a result input was too light. Now it looks fine.

**Comment by hor-kanchan-okta** [comment]
@eugenebakaiev-okta can you update your commit message following 
https://github.com/okta/okta-signin-widget/blob/master/CONTRIBUTING.md

**Comment by rchild-okta** [comment]
FYI @hor-kanchan-okta when merging, you also have an option to update the commit message. So in this case when someone forgets, I think it's good to call it out, but you can also just update it yourself and merge anyway if everything looks okay.

**Pull request merged**